packages/rpc-provider/test/mockHttp.ts

```diff
@@ -4,7 +4,7 @@
 
 import nock from 'nock';
 
-const TEST_HTTP_URL = 'http://localhost:9944';
+const TEST_HTTP_URL = 'http://localhost:9933';
```

Comment:
Should not overlap with the actual node endpoints. (So changing it is correct, changing it ot an overlapping port is not)